.github/workflows/go-build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build(Go)
+run-name: Build (${{ inputs.os-version }}/${{ inputs.go-version }})
+
+on:
+  workflow_call:
+    inputs:
+      os-version:
+        description: 'GitHub-hosted runnners'
+        default: ubuntu-22.04
```

Comment:
I want the default value to be well without maintenance than parameters passed from outside, I want it to be stable. I think `ubuntu-latest` is more stable than `ubuntu-22.04`.